src/i965_drv_video.c

```diff
@@ -2744,14 +2744,18 @@ i965_CreateContext(VADriverContextP ctx,
     i965->current_context_id = contextID;
 
     i965->bufmgr = drm_intel_bufmgr_gem_init(intel->fd, 4096);
-
+    assert(i965->bufmgr);
```

Comment:
Please return error instead of assert if failing to initialize the buffer management